### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by michaelrommel** [michaelrommel/nvim-silicon#18] [Announcements]
## What's changed

squash commit [a724b04](https://github.com/michaelrommel/nvim-silicon/commit/a724b04fea19d9284cf353996831bcf45655be77): 
* feat: added lua functions for easier keymap integration
* feat: simultaneous clipboard and file image support
* feat: WSL2 integration, pushes image to Windows clipboard
* fix: rename lua module to match github repo name

### Previous changes
* feat: add debug and disable_defaults options and improve language detection ([ca6b991](https://github.com/michaelrommel/nvim-silicon/commit/ca6b991142a9526b75fc55a7396e4274e4da606e))
* fix: remove some defaults to allow overriding from opts table ([b744afd](https://github.com/michaelrommel/nvim-silicon/commit/b744afd29f8d7db23cba09e554b316463d526155))
* feat: num_separator setting allows insertion of characters between number margin and code ([6637c3e](https://github.com/michaelrommel/nvim-silicon/commit/6637c3e366bcee54a322672ad57d4697cca9a920))
* fix: move utils package into correct namespace ([383f722](https://github.com/michaelrommel/nvim-silicon/commit/383f722c5b9802ba1d7d99d03a70c49185acf7c9))
* feat: support functions as line_offset option ([e3bb5d3](https://github.com/michaelrommel/nvim-silicon/commit/e3bb5d36b33952cdb3b45faaf0f2853f72ca0c01))

## Contributions 
* feat: add `--window-title` support by @andresatierf in https://github.com/michaelrommel/nvim-silicon/pull/1
* fix: Background Image can not be set as Background color has in Non-nullifiable default by @dangtrivan15 in https://github.com/michaelrommel/nvim-silicon/pull/8
* fix: correct path displayed in notification for generated files by @chee-zaram in https://github.com/michaelrommel/nvim-silicon/pull/13

## New Contributors
* @andresatierf made their first contribution in https://github.com/michaelrommel/nvim-silicon/pull/1
* @dangtrivan15 made their first contribution in https://github.com/michaelrommel/nvim-silicon/pull/8
* @chee-zaram made their first contribution in https://github.com/michaelrommel/nvim-silicon/pull/13

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelrommel/nvim-silicon/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by michaelrommel** [comment]
Hi all,

I released now v1.0.0 as first release with some feature additions and created this discussion thread to get feedback. Some raised issues up to this point have not been implemented, either because of my limited time or because I felt, that adding a feature might overcomplicate the code and I use now my workaround suggestion on a daily basis with good success.

So these discussions or feedback requests to larger audiences will happen here now.

Thanks to all of you, who are using the plugin and gave feedback, what to improve!

  Michael.

**Reply by chee-zaram** [reply]
Congrats on the release, Michael. Happy to have contributed to this minimal and helpful plugin.
